app/assets/stylesheets/application.scss

```diff
@@ -0,0 +1,4 @@
+@import "bootstrap-sprockets";
+@import "theme/variables";
+@import "bootstrap";
+@import "theme/bootswatch";
```

Comment:
I'm hesitant to put the `_variables.scss` in a `/vendor` subdir since that's something we'd likely change often. The `_bootswatch.scss` also might be subjects to updates down the line, should we want to make changes. I'd want to reserve `/vendor` for files we never touch except for updates from the vendor. What do you think @adarsh?